File: src/main/java/me/zodd/postmanpat/mail/MailUserStorage.java
package me.zodd.postmanpat.mail;

import redempt.redlib.config.annotations.Comment;
import redempt.redlib.config.annotations.ConfigMappable;

import java.util.*;

@ConfigMappable
public class MailUserStorage {

    MailUserStorage() {
    }

    @Comment("A map of players ignore-lists, stored by UUID")
    public static Map<UUID, List<UUID>> mailIgnoreList = new HashMap<>(HashMap.newHashMap(3));
}

File: src/main/kotlin/me/zodd/postmanpat/mail/MailUserStorage.kt
package me.zodd.postmanpat.mail

import org.spongepowered.configurate.objectmapping.ConfigSerializable
import java.util.UUID

@ConfigSerializable
data class MailUserStorage(
    val mailIgnoreList: MutableMap<UUID, MutableList<UUID>> = mutableMapOf()
)
